src/containers/my-offers/my-offers-container/MyOffersContainer.constants.tsx

```diff
@@ -0,0 +1,50 @@
+import { useTranslation } from 'react-i18next'
+
+import StatusChip from '~/components/status-chip/StatusChip'
+import SubjectLevelChips from '~/components/subject-level-chips/SubjectLevelChips'
+
+import { getFormatedDate } from '~/utils/helper-functions'
+import { Offer, RemoveColumnRules } from '~/types'
+
+const GetPriceCellValue = (item: Offer) => {
```

Comment:
the component in constants? bad idea, remove it

Reply:
fixed